frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -319,9 +318,9 @@ function FileSearcher:showFileDialog(item)
     if item.is_file then
         local is_currently_opened = self.ui.document and self.ui.document.file == file
         local has_provider = DocumentRegistry:hasProvider(file)
-        local been_opened = BookList.getBookInfoCacheBeenOpened(file)
+        local been_opened = BookList.isBeenOpened(file)
```

Comment:
Wonky grammar ;).

What's the tense you were going for?

isBeingOpened?
hasBeenOpened?

Reply:
Okay, I see I was late to the party, lol ;D.